src/lib.rs

```diff
@@ -1994,6 +1994,7 @@ impl Build {
                         "arm-unknown-linux-gnueabi" => Some("arm-linux-gnueabi"),
                         "armv4t-unknown-linux-gnueabi" => Some("arm-linux-gnueabi"),
                         "armv5te-unknown-linux-gnueabi" => Some("arm-linux-gnueabi"),
+                        "armv5te-unknown-linux-musleabi" => Some("arm-linux-gnueabi"),
```

Comment:
Should this be `arm-linux-musleabi`, or is intentional that gcc is used for the musl target?

Reply:
`*-musleabi` is the same target as `*-gnueabi` but it uses a static linking with [musl libc](https://musl.libc.org/). Both targets use gcc.
I tested some `*-sys` crates and it works fine.
Neither Debian nor Ubuntu have `arm-linux-musleabi` toolchain packages.

Reply:
Ok, thanks for confirming!